addon/-record-data-private/system/store.js

```diff
@@ -49,6 +50,8 @@ const badIdFormatAssertion = '`id` passed to `findRecord()` has to be non-empty
 const emberRun = emberRunLoop.backburner;
 
 const { ENV } = Ember;
+const WeakMapImplementation = WeakMap || Ember.WeakMap;
```

Comment:
`Ember.WeakMap` already uses the global if available, and this would throw an error if it were not available.